tools/walletextension/test/wallet_extension_test.go

```diff
@@ -278,13 +273,12 @@ func TestKeysAreReloadedWhenWalletExtensionRestarts(t *testing.T) {
 //}
 
 func TestGetStorageAtForReturningUserID(t *testing.T) {
-	hostPort := _hostWSPort + _testOffset*8
-	walletHTTPPort := hostPort + 1
-	walletWSPort := hostPort + 2
+	walletHTTPPort := _hostWSPort + 1
+	walletWSPort := _hostWSPort + 2
 
-	createDummyHost(t, hostPort)
-	walExtCfg := createWalExtCfg(hostPort, walletHTTPPort, walletWSPort)
-	createWalExtCfg(hostPort, walletHTTPPort, walletWSPort)
+	createDummyHost(t, _hostWSPort)
+	walExtCfg := createWalExtCfg(_hostWSPort, walletHTTPPort, walletWSPort)
+	createWalExtCfg(_hostWSPort, walletHTTPPort, walletWSPort)
```

Comment:
This still doesn't make a lot of sense, but I didn't want to refactor these methods in this PR.